### DISCUSSION TITLE
Adding support for BDSMLR

### DISCUSSION THREAD

**Discussion by gillesDonogal** [TumblThreeApp/TumblThree#303] [Ideas]
Hello, I wondered if you could add support for the blogging website "bdsmlr.com". It is very close of tumblr, on the way it works.

**Comment by thomas694** [comment]
Hello,
this is a tiny niche blogging site. It has no API and there's no development activity since over a year.
We won't integrate this site.

**Comment by pakiec22** [comment]
Nothing has changed? Still the answer is; We won't integrate this site. ?

**Reply by thomas694** [reply]
Not much, but nothing is carved in stone.
Everyone who would like to download from that site, give the topic an up arrow.

**Comment by Someonemustnothavethis** [comment]
I keep a page on BDSMLR and it's complete garbage.  Nothing works except the horde of intrusive ads and there's been no improvement in years.  Don't ask about the complete lack of news.  Can't see how this place survives.  Please don't waste your time with it.
